uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document.controller.spec.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env amd, jasmine */
```

Comment:
It only tests that the controller is there and if it compiles, but no other tests. What was the initial plan?

Reply:
Oh, sorry, I deleted one of them tests when I force pushed!!!

Reply:
right. apologies.